BugFix: Dump query

locke-lunarg opened the pull request:
It has to ensure that the command list has a pair of BeginQuery and EndQuery.

davidd-lunarg commented on framework/graphics/dx12_dump_resources.cpp:
```diff
@@ -35,7 +35,7 @@ GFXRECON_BEGIN_NAMESPACE(graphics)
 // But here uses fixed type.
 // According to the resource's desc.Dimension, float is for buffer, image is for the others.
-const bool TEST_READABLE = false;
+const bool TEST_READABLE = true;
```
Is this supposed to be always on now?

locke-lunarg replied:
Sorry, No. Fixed.

davidd-lunarg reviewed (approved):
Looks good!

locke-lunarg committed:
dx12 dump: Bugfix Query

It has to ensure that the splited command list has a pair of BeginQuery and EndQuery.
